api/proto/protoserver/protoserver.go

```diff
@@ -0,0 +1,39 @@
+package protoserver
+
+import (
+	"github.com/hashicorp/raft"
+	"google.golang.org/grpc"
+	"net"
+	"nubedb/api/proto"
+	"nubedb/cluster/consensus/fsm"
+	"nubedb/internal/app"
+	"nubedb/internal/config"
+)
+
+type server struct {
+	proto.UnimplementedServiceServer
+	Config    config.Config
+	Consensus *raft.Raft
+	FSM       *fsm.DatabaseFSM
+}
+
+func Start(a *app.App) error {
+	listen, errListen := net.Listen("tcp", a.Config.CurrentNode.GrpcAddress)
+	if errListen != nil {
+		return errListen
+	}
+	srvModel := new(server)
+
+	protoServer := grpc.NewServer()
+	proto.RegisterServiceServer(protoServer, srvModel)
+	srvModel.Config = a.Config
```

Comment:
Before this was bellow server and got into race conditions, maybe this should be done better

Reply:
Done in: https://github.com/narvikd/nubedb/pull/10/commits/53669c68d0141d3cebae767df3193d9b27d4d548